lammps-tutorials.tex

```diff
@@ -4230,10 +4241,9 @@ \section*{Potentially Conflicting Interests}
 \section*{Funding Information}
```

Comment:
@jrgissing If you have a funding that you feel is relevant, don't hesitate to add it.